tests/src/test/scala/whisk/core/limits/ActionLimitsTests.scala

```diff
@@ -245,7 +245,7 @@ class ActionLimitsTests extends TestHelpers with WskTestHelpers {
     val allowedSize = ActivationEntityLimit.MAX_ACTIVATION_ENTITY_LIMIT.toBytes
 
     // Needs some bytes grace since activation message is not only the payload.
-    val args = Map("p" -> ("a" * (allowedSize - 700).toInt).toJson)
+    val args = Map("p" -> ("a" * (allowedSize - 750).toInt).toJson)
```

Comment:
? this change seems unrelated

Reply:
Not really. Serializing the uuid as an additional part of the namespace has added some crucial bytes to the message so that the "grace bytes" where not enough to get the message passed.